easybuild/easyconfigs/p/PAGIT/PAGIT-V1.64bit.eb

```diff
@@ -0,0 +1,41 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Tarball'
+
+name = 'PAGIT'
+version = 'V1'
+versionsuffix = '.64bit'
+
+homepage = 'http://www.sanger.ac.uk/science/tools/pagit'
+description = """PAGIT addresses the need for software to generate high quality draft genomes"""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['ftp://ftp.sanger.ac.uk/pub/resources/software/pagit/']
+sources = ['%(name)s.%(version)s%(versionsuffix)s.tgz']
+
+osdependencies = ['tcsh']
+
+modextrapaths = {
+       'PATH': "bin:$root/PAGIT/bin:$root/PAGIT/bin/pileup_v0.5/:$root/PAGIT/bin/pileup_v0.5/ssaha2:$root/PAGIT/IMAGE/:$root/PAGIT/ABACAS:$root/PAGIT/ICORN:$root/PAGIT/RATT",
+       'PERL5LIB': 'PAGIT/lib',
+}
+
+modextravars = {
+    'PAGIT_HOME':'$root/PAGIT',
```

Comment:
Thanks for mentioning that, I've enhanced the easyconfig tests to catch this: https://github.com/hpcugent/easybuild-easyconfigs/pull/4330